chain_info/Astar.yaml

```diff
@@ -0,0 +1,53 @@
+name: Astar
+network_type: mainnet
+chain:
+  type: parachain
+  parent: Polkadot
+  consensus: POS
+  sudo: true
+  para_id: 2006
+chainspec:
+  http_url: https://raw.githubusercontent.com/AstarNetwork/Astar/master/bin/collator/res/astar.json
+goals:
+  - The Future of Smart Contracts for Multichain
+repository: https://github.com/AstarNetwork/Astar
+specs:
+  block_time: 12
+  ed: 1000000
+  decimals: 18
+  token: ASTR
+  ss58_format: 5
+contacts:
+  - type: email
+    contact: info@astar.network
+documentation:
+  - https://docs.astar.network/
+validators: []
+release_cycle: []
+rpc_endpoints:
+  - name: BlastAPI (WSS)
+    url: https://astar.public.blastapi.io
+  - name: Dwellir (WSS)
+    url: https://astar-rpc.dwellir.com

```

Comment:
this should be `wss` not `https`

Reply:
We have both since we support traditional RPC for EVM. WSS ones are listed below. Do you suggest I remove these completely?

Reply:
no, my comment was about the protocol and the name mismatch.
you have this `BlastAPI (WSS)` but the `url` contains `https://` and not `wss://`

So one of this is wrong, the `name` or the `url`

Reply:
Gotcha. Apologies, too much copy paste. Fixed.